BannerlordTwitch/BLTAdoptAHero/_Module/ModuleData/Languages/DE/strings-behaviors-DE.xml

```diff
@@ -1,193 +1,268 @@
+            original="Can't upgrade retinue any further!" />
         <!-- was murdered -->
-        <string id="LhHul2lV" text="wurde ermordet" original="was murdered"/>
+        <string id="LhHul2lV" text="wurde ermordet" original="was murdered" />
         <!-- died in labor -->
-        <string id="HwjR45XN" text="starb bei der Arbeit" original="died in labor"/>
+        <string id="HwjR45XN" text="starb bei der Arbeit" original="died in labor" />
         <!-- died of old age -->
-        <string id="5GOjfkzW" text="an Altersschwäche gestorben" original="died of old age"/>
+        <string id="5GOjfkzW" text="an Altersschwäche gestorben" original="died of old age" />
         <!-- died in battle -->
-        <string id="ZKrgqWav" text="starb im Kampf" original="died in battle"/>
+        <string id="ZKrgqWav" text="starb im Kampf" original="died in battle" />
         <!-- was wounded in battle -->
-        <string id="jp4sldTL" text="wurde im Kampf verwundet" original="was wounded in battle"/>
+        <string id="jp4sldTL" text="wurde im Kampf verwundet" original="was wounded in battle" />
         <!-- was executed -->
-        <string id="SkFFXsI1" text="wurde ausgeführt" original="was executed"/>
+        <string id="SkFFXsI1" text="wurde enthauptet" original="was executed" />
         <!-- was lost -->
-        <string id="HMHdXDaK" text="war verloren" original="was lost"/>
+        <string id="HMHdXDaK" text="ist verschollen" original="was lost" />
         <!-- was ended -->
-        <string id="lrOJnThZ" text="war beendet" original="was ended"/>
+        <string id="lrOJnThZ" text="wurde ein Ende bereitet" original="was ended" />
         <!-- BLTTournamentBetMissionBehavior -->
         <!-- Betting is now OPEN for round {RoundIndex} match: {Teams}! -->
-        <string id="wgGkXkyh" text="Das Wetten ist jetzt für Runde {RoundIndex} match: {Teams} geöffnet!" original="Betting is now OPEN for round {RoundIndex} match: {Teams}!"/>
+        <string id="wgGkXkyh" text="Wetten für Runde {RoundIndex} match: {Teams} sind jetzt OFFEN!"
+            original="Betting is now OPEN for round {RoundIndex} match: {Teams}!" />
         <!-- Betting is now OPEN for final match: {Teams}! -->
-        <string id="9BjHfivM" text="Wetten sind jetzt OFFEN für das letzte Spiel: {Teams}!" original="Betting is now OPEN for final match: {Teams}!"/>
+        <string id="9BjHfivM" text="Wetten sind jetzt OFFEN für das letzte Spiel: {Teams}!"
+            original="Betting is now OPEN for final match: {Teams}!" />
         <!-- vs -->
-        <string id="ixTCiaiv" text="vs" original="vs"/>
+        <string id="ixTCiaiv" text="vs" original="vs" />
         <!-- blue -->
-        <string id="HrIifwJy" text="Blau" original="blue"/>
+        <string id="HrIifwJy" text="blue" original="blue" />
         <!-- red -->
-        <string id="6ZGgsbe1" text="rot" original="red"/>
+        <string id="6ZGgsbe1" text="red" original="red" />
```

Comment:
@billw2012  It did not work with the translation. We tried `rot` and `Rot` (`!bltbet rot 123` or `!bltbet Rot 123`). Therefore we changed it back to the english names so we can use tournament bets with German.

Reply:
This is strange, I'm playing in Russian and using `ru` translation it works just fine

Reply:
I'm not a streamer, otherwise I would create a video proof but I noticed it from 3 different German streamers, that it's not working with German translation.